bin/circle/compute-failure-rates

```diff
@@ -97,10 +101,17 @@ end
 
 # Print build stats
 puts "Failure rates for builds #{build_range} by date (1.0 = 100% failures)"
-puts "%-10s#{' %16s' * columns.size}" % (['Date'] + columns.keys.map {|k| k[0, 14]})
-stats_by_date.each do |date, stats|
-  puts "%-10s#{'  %7s = %5.3f' * columns.size}" %
+if options[:output_format] == 'csv'
+  puts "%s#{',%s' * columns.size}" % (['Date'] + columns.keys)
```

Comment:
For future reference there's also a stdlib [CSV::Writer](https://ruby-doc.org/stdlib-1.8.7/libdoc/csv/rdoc/CSV/Writer.html), though this is simple enough!